{ 
  "text" : "libxml2 2.9.8, if --with-lzma is used, allows remote attackers to cause a denial of service (infinite loop) via a crafted XML file that triggers LZMA_MEMLIMIT_ERROR, as demonstrated by xmllint, a different vulnerability than CVE-2015-8035 and CVE-2018-9251.\n" ,
  "entities" : [ { 
    "eid" : "E1" ,
    "keyid" : "ThreatActor#d0f531219210b35f7261f3d6049ee44c" ,
    "type" : "ThreatActor" ,
    "subType" : "ThreatActor" ,
    "value" : "remote attacker" ,
    "begin" : 46 ,
    "end" : 62 ,
    "canonicalForm" : "remote attacker" ,
    "score" : "0.0"
  }
  , { 
    "eid" : "E2" ,
    "keyid" : "TTP#9c98ef6a7451d6f82ca2194ee23d1acb" ,
    "type" : "TTP" ,
    "subType" : "TTP" ,
    "value" : "denial of service" ,
    "begin" : 74 ,
    "end" : 91 ,
    "canonicalForm" : "denial of service" ,
    "score" : "0.0"
  }
  , { 
    "eid" : "E3" ,
    "keyid" : "Vulnerability#c81aa7650c5639f22e7f3fc4969a9a12" ,
    "type" : "Vulnerability" ,
    "subType" : "CVE" ,
    "value" : "cve-2015-8035" ,
    "begin" : 225 ,
    "end" : 238 ,
    "canonicalForm" : "CVE-2015-8035" ,
    "score" : "0.0"
  }
  , { 
    "eid" : "E4" ,
    "keyid" : "Vulnerability#013bd667f66513332235e0503fd0c49c" ,
    "type" : "Vulnerability" ,
    "subType" : "CVE" ,
    "value" : "cve-2018-9251" ,
    "begin" : 243 ,
    "end" : 256 ,
    "canonicalForm" : "CVE-2018-9251" ,
    "score" : "0.0"
  }
  ,  ] ,
  "relations" : [ { 
    "rid" : "R1" ,
    "type" : "relatedEntity" ,
    "predicate" : { } ,
    "arguments" : [ { 
      "argnum" : 1 ,
      "begin" : 46 ,
      "end" : 62 ,
      "eid" : "E1" ,
      "canonicalForm" : "remote attacker"
    }
    , { 
      "argnum" : 2 ,
      "begin" : 243 ,
      "end" : 256 ,
      "eid" : "E4" ,
      "canonicalForm" : "CVE-2018-9251"
    }
    ,  ]
  }
  , { 
    "rid" : "R2" ,
    "type" : "cause" ,
    "predicate" : { 
      "begin" : 63 ,
      "end" : 71 ,
      "surfaceForm" : "to cause"
    }
     ,
    "arguments" : [ { 
      "argnum" : 1 ,
      "begin" : 46 ,
      "end" : 62 ,
      "eid" : "E1" ,
      "canonicalForm" : "remote attacker"
    }
    , { 
      "argnum" : 2 ,
      "begin" : 74 ,
      "end" : 91 ,
      "eid" : "E2" ,
      "canonicalForm" : "denial of service"
    }
    ,  ]
  }
  , { 
    "rid" : "R3" ,
    "type" : "relatedEntity" ,
    "predicate" : { } ,
    "arguments" : [ { 
      "argnum" : 1 ,
      "begin" : 74 ,
      "end" : 91 ,
      "eid" : "E2" ,
      "canonicalForm" : "denial of service"
    }
    , { 
      "argnum" : 2 ,
      "begin" : 243 ,
      "end" : 256 ,
      "eid" : "E4" ,
      "canonicalForm" : "CVE-2018-9251"
    }
    ,  ]
  }
  , { 
    "rid" : "R4" ,
    "type" : "relatedEntity" ,
    "predicate" : { } ,
    "arguments" : [ { 
      "argnum" : 1 ,
      "begin" : 74 ,
      "end" : 91 ,
      "eid" : "E2" ,
      "canonicalForm" : "denial of service"
    }
    , { 
      "argnum" : 2 ,
      "begin" : 225 ,
      "end" : 238 ,
      "eid" : "E3" ,
      "canonicalForm" : "CVE-2015-8035"
    }
    ,  ]
  }
  , { 
    "rid" : "R5" ,
    "type" : "relatedEntity" ,
    "predicate" : { } ,
    "arguments" : [ { 
      "argnum" : 1 ,
      "begin" : 225 ,
      "end" : 238 ,
      "eid" : "E3" ,
      "canonicalForm" : "CVE-2015-8035"
    }
    , { 
      "argnum" : 2 ,
      "begin" : 243 ,
      "end" : 256 ,
      "eid" : "E4" ,
      "canonicalForm" : "CVE-2018-9251"
    }
    ,  ]
  }
  , { 
    "rid" : "R6" ,
    "type" : "relatedEntity" ,
    "predicate" : { } ,
    "arguments" : [ { 
      "argnum" : 1 ,
      "begin" : 46 ,
      "end" : 62 ,
      "eid" : "E1" ,
      "canonicalForm" : "remote attacker"
    }
    , { 
      "argnum" : 2 ,
      "begin" : 225 ,
      "end" : 238 ,
      "eid" : "E3" ,
      "canonicalForm" : "CVE-2015-8035"
    }
    ,  ]
  }
  ,  ]
}
